app/views/users/show.html.erb

```diff
@@ -1,2 +1,22 @@
-<h1>Users shown per id</h1>
-<p>Find me in app/views/users/show.html.erb</p>
+<%= render @user %>
+
+<ul class="user-main">
+<li class="user-bio">
+    <h3>Bio : </h3>
+    <p><%= @user.bio%></p>
+</li>
+
+
+   <% @user.recent_posts.each_with_index do |post, i|%>
+        <li class="post-all">
+        <h4>Post #<%=i+1%></h4>
+        <p><%= post.text%></p>
+        <p class="show-li">Coments : <%= post.comments_counter.to_i%> Likes : <%= post.likes_counter.to_i %> </p>
+        </li>
+    <%end%>
+</ul>
+
+
+<button class='btn'>
```

Comment:
- [ ] Good job working on this but when I click a user to view the show page I get an error. Please kindly fix this error.

![image](https://user-images.githubusercontent.com/43995830/215070459-013a9a89-3722-4ae1-9f6b-f918fd2ba645.png)